openlibrary/core/sponsorships.py

```diff
@@ -41,7 +41,11 @@ def get_sponsorable_editions():
     """This should move to an infogami type so any admin can add editions
     to the list. This will need to be paginated.
     """
-    candidates = web.ctx.site.get('/sponsorship/books').get('editions')
+    try:
+        candidates = web.ctx.site.get('/sponsorship/books').get('editions')
```

Comment:
Pythonic way to do this would be:

```py
candidates = web.ctx.site.get('/sponsorship/books').get('editions', [])
```

Then you don't need the `try/except`; but I'm not 100% which `.get` is throwing the `AttributeError` though, so it might not work.